### DISCUSSION TITLE
2FA via email/PC

### DISCUSSION THREAD

**Discussion by uhilgert** [community/community#78721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I seem to be unable to figure out how to initiate 2FA authentication using my email account. I am using GitHub exlusively for work (teaching workshops) and am not interested in using my personal mobile phone for this purpose. Thank you for any advise you could provide.

**Comment by DylanDevelops** [comment]
To my knowledge, there isn't a way to do this, but you can view what options there are for 2F authentication in your `Account Settings` > `Password and authentication` > `Two-factor authentication` > `Preferred 2FA method`.

**Comment by DNin01** [comment]
While authenticator (a.k.a. TOTP) apps usually reside on mobile devices, there are options available for desktop platforms. Some password managers such as [KeePassXC](https://keepassxc.org/) also have TOTP functionality.

**Comment by PauloDC** [comment]
This is a very common error: To create a SINGLE POINT OF FAILURE. If a working mobile device is the only option for 2FA, it implies that every time my mobile device is not working I lose access. And I don't need to mention all reasons why a specific working mobile device might be unavailable.
The solution to avoid this common error is to support an alternative. Most commonly, the email alternative.
